src/pools/abstract-pool.ts

```diff
@@ -152,12 +143,12 @@ export abstract class AbstractPool<
    *
    * @param numberOfWorkers Number of workers that this pool should manage.
    * @param filePath Path to the worker-file.
-   * @param opts Options for the pool. Default: `{ maxTasks: 1000 }`
+   * @param opts Options for the pool. Default: `{}`
    */
   public constructor (
     public readonly numberOfWorkers: number,
     public readonly filePath: string,
-    public readonly opts: PoolOptions<Worker> = { maxTasks: 1000 }
+    public readonly opts: PoolOptions<Worker> = {}
```

Comment:
@Shinigami92  I would remove this default since that any implementation of the worker already has a default value so this is redundant and is not covered also.
Or we remove it from the abstract pool, or we remove it from the other implementations.
My preference is to remove it from the abstract pool so that any implementation can have different defaults in the future if needed

Reply:
Okay, but then please also remove the
```
Default: `{}`
```
from the JSDoc

Reply:
k, done